Shared_UI/Shared_UI.xcodeproj/project.pbxproj

```diff
@@ -148,7 +154,7 @@
 					};
 				};
 			};
-			buildConfigurationList = B62D386529BA635400179020 /* Build configuration list for PBXProject "Shared_UI" */;
+			buildConfigurationList = B62D386529BA635400179020 /* Build configuration list for PBXProject "shared_UI" */;
```

Comment:
why this change

Reply:
i didn't make this change! , how to fix it

Reply:
you can change on this file manual